README.md

```diff
@@ -328,6 +328,86 @@ This process is inspired by Bitcoin. The difference is that we use a single SHA3
   did:abt:zNKtCNqYWLYWYW3gWRA1vnRykfCBZYHZvzKr
   ```
 
+### Declare DID
+
+Declaring a DID is done by sending a declare transaction to the ABT network. The following is a sample transaction.
```

Comment:
ABT Network --> Blockchain, since it's not necessary to be ABT Network, could be any forgeframework powered chain